### ISSUE TITLE
Opera 15+ suggestion (from Hacker News)

### ISSUE THREAD

**Issue by ghost** [bleachbit/bleachbit#127]
Use Chrome XML for Opera above 15:
 https://news.ycombinator.com/item?id=11231673
 https://news.ycombinator.com/item?id=11231305

**Comment by az0** [comment]
Interesting. I will look into it.

Meanwhile, perhaps you want to do a search and replace on Chromium.xml and
start testing?

**Comment by vmorgulys** [comment]
I discovered the console version. Very cool :-)

Here is my XML file:

[opera_cef_xml.txt](https://github.com/az0/bleachbit/files/160527/opera_cef_xml.txt)

I just changed the id and replaced "Chromium" by "Opera" (and "chromium" by "opera"). I left the ".cxchromium" (don't have that here). I used the latest (github) chromium file. I run debian and the XML in /usr/share/bleachbit/cleaners is a bit older.

An output from the command line (with french messages):

```
valar@winter:~$ bleachbit --preview operacef.*
Gtk-Message: Failed to load module "canberra-gtk-module"
debug: '/usr/share/bleachbit/cleaners/localizations.xml' is not usable
debug: clean_operation('operacef'), options = '[u'cache', u'cookies', u'current_session', u'dom', u'form_history', u'history', u'passwords', u'search_engines', u'vacuum']'
Supprimer 94,2kB /home/valar/.cache/opera/Cache/data_2
Supprimer 131,1kB /home/valar/.cache/opera/Cache/f_00000a
Supprimer 24,6kB /home/valar/.cache/opera/Cache/f_000004
Supprimer 53,2kB /home/valar/.cache/opera/Cache/f_000005
Supprimer 28,7kB /home/valar/.cache/opera/Cache/f_000009
Supprimer 225,3kB /home/valar/.cache/opera/Cache/index
Supprimer 49,2kB /home/valar/.cache/opera/Cache/f_000008
Supprimer 24,6kB /home/valar/.cache/opera/Cache/f_000003
Supprimer 36,9kB /home/valar/.cache/opera/Cache/f_00000b
Supprimer 53,2kB /home/valar/.cache/opera/Cache/f_00000c
Supprimer 53,2kB /home/valar/.cache/opera/Cache/f_00000d
Supprimer 81,9kB /home/valar/.cache/opera/Cache/f_000002
Supprimer 45,1kB /home/valar/.cache/opera/Cache/data_1
Supprimer 12,3kB /home/valar/.cache/opera/Cache/data_0
Supprimer 41kB /home/valar/.cache/opera/Cache/f_000001
Supprimer 20,5kB /home/valar/.cache/opera/Cache/f_000006
Supprimer 32,8kB /home/valar/.cache/opera/Cache/f_000007
Supprimer 675,8kB /home/valar/.cache/opera/Cache/data_3
Nettoyer ce fichier ?B /home/valar/.config/opera/Local State
Nettoyer ce fichier ?B /home/valar/.config/opera/Local State
Espace disque potentiellement récupérable : 1,7MB
Fichiers potentiellement effaçables : 18
Opérations spécifiques : 2

```

I ran a clean and seen the cache flushed.

**Comment by az0** [comment]
To do item

- [ ] check into extension cookies

**Comment by az0** [comment]
The fix was published in [BleachBit 2.1 beta](https://www.bleachbit.org/news/bleachbit-21-beta). Please try it out, and let me know how it goes. 

Soon the fix will be published also in BleachBit 2.2 final.
